.jenkinsci/debug-build.groovy

```diff
@@ -48,19 +59,24 @@ def doDebugBuild(coverageEnabled=false) {
     """  
     sh """
       cmake \
-        -DCOVERAGE=ON \
         -DTESTING=ON \
         -H. \
         -Bbuild \
         -DCMAKE_BUILD_TYPE=Debug \
-        -DIROHA_VERSION=${env.IROHA_VERSION}
+        -DIROHA_VERSION=${env.IROHA_VERSION} \
+        ${cmakeOptions}
     """
     sh "cmake --build build -- -j${parallelism}"
     sh "ccache --show-stats"
-    sh "cmake --build build --target test"
-    sh "cmake --build build --target cppcheck"
-
     if ( coverageEnabled ) {
+      sh "lcov -i --capture --directory build --config-file .lcovrc --output-file build/reports/coverage.init.info"
```

Comment:
May you explain the purpose of that line?

Reply:
According to `lcov` manual, we have to generate so-called 'baseline' report prior running tests. That is to ensure proper line coverage percentage. [See lcov man page](http://ltp.sourceforge.net/coverage/lcov/lcov.1.php)